.github/workflows/abi-report.yml

```diff
@@ -28,7 +28,7 @@ permissions:
 
 jobs:
   check:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Should we not specify latest and always use a specific version on all CI?

Reply:
I definitely prefer that, but ubuntu-latest is 22.04, which does not have gcc-14